apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -162,7 +162,10 @@ const BookingFields = ({
         }
 
         const label = noLabel ? "" : field.label || t(field.defaultLabel || "");
-        const placeholder = field.placeholder || t(field.defaultPlaceholder || "");
+        const placeholder =
```

Comment:
NIT: prefered `includes` over `indexOf`, it should be consistent throughout the codebase whenever comparing values

https://handbook.cal.com/engineering/best-practices/avoid-comparing-multiple-values-with-includes

Reply:
This isn't the right place to do it. You should remove the defaultPlaceholder here https://github.com/calcom/cal.com/blob/7271bae91ade4055829e31cf7e1e5f1d6c4a777f/packages/features/bookings/lib/getBookingFields.ts#L166

Reply:
@hariombalhara If I read that correctly, these `systemBeforeFields`are used in other instances beyond the book page. Should the placeholders then be removed in all of those instances or specifically on the book page?

Reply:
Yeah, that is the single source of Truth for Name and Email fields. This would also allow the placeholder to go away from Manage Booking Questions - UI as well.
<img width="1400" alt="Screenshot 2023-03-14 at 7 26 13 PM" src="https://user-images.githubusercontent.com/1780212/225023871-bd2c4aab-f0f6-4569-8bf5-be70421e6491.png">